loadtest/control/simulcontroller/actions.go

```diff
@@ -22,6 +22,8 @@ import (
 type userAction struct {
 	run       control.UserAction
 	frequency int
+	// Minimum supported server version
+	minServerVersion string
```

Comment:
I might be missing something, but where are we actually setting this field?

Reply:
You are not missing anything. I am just not setting it on existing actions since they have been there since the start and they should work on any recent version. My idea would be to support until [ESR](https://docs.mattermost.com/administration/extended-support-release.html) and they clearly met that. Also tracking down which actions map to which version might be hard to do retroactively so if the field is not set the default is that the action will be run regardless of the server version.

If a new action were to be added then we should be setting its `minServerVersion` field somewhere here:

https://github.com/mattermost/mattermost-load-test-ng/blob/cb98a18ef1fa6bf16142fb6406c7a7ad1f11f5b6/loadtest/control/simulcontroller/controller.go#L109-L178